src/include/ikarus/utils/observer/nonLinearSolverLogger.hh

```diff
@@ -35,11 +35,12 @@ public:
         iters = 0;
         dNorm = 0.0;
         rNorm = 0.0;
+        sNorm = 0.0;
         spdlog::info("Non-linear solver started:");
```

Comment:
Maybe we don't use this observer and directly write these statements inside the NewtonSolver